src/extensions/extension_manager/inc/aduc/extension_manager.hpp

```diff
@@ -21,8 +21,8 @@
 #include <string>
 #include <unordered_map>
 
-// Default DO retry timeout is 24 hours.
-#define DO_RETRY_TIMEOUT_DEFAULT (60 * 60 * 24)
+// Default Content Downloader Maximum Timeout in seconds before the downloader gives up trying to download.
```

Comment:
do we really want to make the default timeout 24 hours?  that's an incredibly long time unless someone is over a satellite connection or something.

Reply:
Made it 8 hours and in minutes (within extension manager domain).  It still has to stay as seconds units b/c that's the content download interface contract, so I convert it.